local-cli/__tests__/bundle-test.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+var Promise = require('promise');
+var Bundle = require('../../packager/react-packager/src/Bundler/Bundle');
+
+jest.mock('fs');
+jest.dontMock('../bundle');
+jest.dontMock('path');
+
+jest.setMock('../../packager/react-packager', {
```

Comment:
Hmm not sure why it doesn't work. Either ways you'd still need to have pretty much all this code to return the promise with the bundle